crowdloans/projects/kusama.json

```diff
@@ -399,5 +399,14 @@
         { "name": "Crowdloan", "value": 10, "color": "#ffbd71" },
         { "name": "Early Contributors", "value": 18, "color": "#f9e07f" }
       ]}
+  },
+  {
+    "paraid": "2126",
+    "name": "Parallel Heiko",
+    "token": "HKO",
+    "description": "Decentralized money market protocol that offers lending, staking, and borrowing",
```

Comment:
Decentralized DeFi protocol that offers a suite of DeFi products from lending, borrowing, liquid staking, AMM, and farming.